fix(curriculum): Updated step 74 of spreadsheet project and added step 75 and step 76 (#54417)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/661f49650572031c6ebdb8e3.md

---
id: 661f49650572031c6ebdb8e3
title: Step 76
challengeType: 0
dashedName: step-76
---

# --description--

> Remove line code line `return regex.test(str);` and line `console.log(highPrecedence("5*3"))` as they were only for testing purposes.

Reviewer: ```suggestion
Remove both the `console.log()` with your `highPrecedence` call, and the `return` statement from your `highPrecedence` function.
```

# --hints--

> Remove the line `console.log(highPrecedence("5*3"))`.

Reviewer: ```suggestion
Your code should not log the result of `highPrecedence("5*3")`.
```

> ```js
> 
> assert.notMatch(code, /console\.log\(highPrecedence\("5\*3"\)\)/);
> 
> ```

Reviewer: ````suggestion
```js
assert.notMatch(code, /console\.log\(highPrecedence\("5\*3"\)\)/);
```
````

> Remove the line `return regex.test(str);`.
> 
> ```js
> 
> assert.notMatch(code, /return\s+regex\.test\(str\);/);
> 
> ```

Reviewer: ````suggestion
Your `highPrecedence` function should not return a value.

```js
assert.isUndefined(highPrecedence("5*3"));
```
````
